integration/tests/cook/test_cli.py

```diff
@@ -1309,6 +1309,37 @@ def test_show_disk(self):
                 self.assertIn("Disk Limit", cli.stdout(cp))
                 self.assertIn("Disk Type", cli.stdout(cp))
 
+    def test_submit_with_disk(self):
```

Comment:
added a new test for the functionality with `cs submit` but another option is just modifying the current test (above) that is used for `cs show`. what do you think?

Reply:
I think it's fine to have a separate one